modules/bigtable-instance/README.md

```diff
@@ -68,10 +66,52 @@ module "bigtable-instance" {
 # tftest modules=1 resources=4 inventory=columns.yaml
 ```
 
-### Instance with replication enabled
+### Instance with table and authorized view
 
 ```hcl
+module "bigtable-instance" {
+  source     = "./fabric/modules/bigtable-instance"
+  project_id = "my-project"
+  name       = "instance"
+  clusters = {
+    my-cluster = {
+      zone = "europe-west1-b"
+    }
+  }
+  tables = {
+    test = {
+      split_keys = ["a", "b", "c"]
+      column_families = {
+        cf1 = {}
+        cf2 = {}
+        cf3 = {}
+      }
+    }
```

Comment:
```suggestion
```
Shouldn't `authorized_views` be under the table in the structure?

Then I get failure in example, that `deletion_protection` is necessary (which looks good, according to variables.tf